### PR TITLE
Fix Issue #1095 : TextInput suggestions appears on unfocused TextInput

### PR DESCRIPTION
**Pull request by Julien-Molina**
See Issue #1095 for more details

image

### CODE REVIEW

File: src/js/components/TextInput.js [hunk 1]
```diff
@@ -236,6 +237,10 @@ export default class TextInput extends Component {
           this._onAddDrop(event);
         }
       }
+
+      if(event.keyCode === tab) {
+        this.setState( { focused: false });
```
Comment (alansouzati) [inline src/js/components/TextInput.js]: totally minor, would you mind removing the extra space before the curly bracket ? We should add that to our linter :)

`this.setState({ focused: false });`

Comment (Julien-Molina) [inline src/js/components/TextInput.js]: Done :)

Comment (Julien-Molina) [inline src/js/components/TextInput.js]: Done :)

**Comment by alansouzati** [comment]
Thanks and welcome to the team

**Pull request merged**